commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java

```diff
@@ -39,16 +40,24 @@
 
 	@Override
 	public CPDefinitionInventory addCPDefinitionInventory(
-			long cpDefinitionId, String cpDefinitionInventoryEngine,
-			String lowStockActivity, boolean displayAvailability,
-			boolean displayStockQuantity, int minStockQuantity,
-			boolean backOrders, int minOrderQuantity, int maxOrderQuantity,
-			String allowedOrderQuantities, int multipleOrderQuantity,
-			ServiceContext serviceContext)
+			long groupId, long cpDefinitionId,
```

Comment:
@igorbeslic you can obtain the groupId from the CPDefinition

Reply:
@marco-leo do you think we can change serviceContext.getUuID with something else? cpDefinition.getUuid() or even generate new uuid? I ask to se if we need serviceContext at all??

Reply:
@igorbeslic even if I don't like that is the portal standard, and I would like to be complainant to it. So when the UUID is needed, we do have to use it for now